config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -516,11 +516,10 @@
          files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]nowhitespaceafter.xml.template"/>
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]nowhitespacebefore.xml.template"/>
-  <!-- until https://github.com/checkstyle/checkstyle/issues/13867 -->
-  <suppress id="propertiesMacroMustExist"
-         files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]typecastparenpad.xml.template"/>
   <suppress id="propertiesMacroMustExist"
          files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]parenpad.xml.template"/>
+  <suppress id="propertiesMacroMustExist"
+         files="src[\\/]xdocs[\\/]checks[\\/]whitespace[\\/]whitespacearound.xml.template"/>
```

Comment:
Why we put it here
https://github.com/checkstyle/checkstyle/blob/63b9e0b71097538b0a1ed59d32108f923cf934d1/src/xdocs/checks/whitespace/whitespacearound.xml.template#L66

Already migrated